packages/utils/src/string.js

```diff
@@ -32,3 +33,10 @@ export const getArticle = word => {
   }
   return word.match(/^[aeiou]/i) ? 'an' : 'a';
 };
+
+const truncate = (string, length) => (string.length <= length ? string : string.slice(0, length));
+
+export const stringToFilename = string => {
```

Comment:
Interesting piece of PR cross pollination here... @tcaiger you may be interested to know that @kael89 just added a `utils/src/filesystem.js` file which has a few functions for dealing with I/O (https://github.com/beyondessential/tupaia/pull/2982/files#diff-c309acd37e4d0905cc5faa87d21a46e5751fbbb6c8e115696c91197709bcda99). I feel this function _may_ be slightly more at home there (and rename to just `toFilename()`) but it's much of a muchness 👍

Reply:
Great work picking up cross-PR similarities @rohan-bes ! 👏

Reply:
Great suggestion!